src/renderer/Generics/redesign/HeaderMetrics/HeaderMetrics.tsx

```diff
@@ -8,10 +8,12 @@ import { NodeOverviewProps } from '../consts';
 export const HeaderMetrics = (props: NodeOverviewProps) => {
   const { screenType, status, stats, rpcTranslation } = props;
   const metricTypeArray: {
+    nodePackage: MetricStats[];
     altruistic: MetricStats[];
     client: MetricStats[];
     validator: MetricStats[];
   } = {
+    nodePackage: ['status', 'memoryUsagePercent', 'cpuLoad', 'diskUsageGBs'],
```

Comment:
on the design, it seems like `memoryUsagePercent` here should be `currentSlot`, but personally, `memoryUsagePercent` makes more sense to me 🤔 maybe confirm with Danny?

Reply:
Ok the design has `latest block` at multi client level which is Execution layer, but `slot` is for consensus layer. I'm ok with showing latest block. I chose memoryUsage as a filler because it was easier to calculate instead of pulling in a value from an RPC call. I may want to kick this to a later pr. @ddannehh

Reply:
We're talking about the package (CC+EC) overview screen here right? The idea is to have the most valuable KPI's for you CC+EC combo available on this screen. The primary stats to determine if your nodes are healthy and working as they should.
If memory usage is a better indicator than current block/slot (not sure which one is more leading of the two or more generalised when talking about the combo as a whole) we should use that one. If one of the KPI's feels off the operator will naturally dig deeper and open EC/CC client detail pages.
Further down the line we could allow the operator to customise which KPI's they want visible perhaps resulting in multiple rows. 🤓

Reply:
Thanks for the explanation. I'm okay if we want to make this latest block in near future. Let's revisit after this, not a blocker imo.